### PR TITLE
fix doc things

### PR DESCRIPTION
**Pull request by orbeckst**
## Description
- fix link to GPL v3 in docs
- fix math type setting mean curvature

### CODE REVIEW

**Comment by IAlibay** [comment]
Note, setup.py claims MIT, probably worth checking for consistency in the license.

**Comment by orbeckst** [comment]
setup.py needs changing

**Comment by orbeckst** [comment]
Well, the LICENSE file _is_ MIT https://github.com/MDAnalysis/membrane-curvature/tree/fix-license-link ... I'll raise a issue.

File: membrane_curvature/curvature.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 Mean curvature is defined by
 
 .. math:: H =
-    \frac{(1+\partial_x^2)\partial_{yy}+(1+\partial_y^2)\partial_{xx}-2\partial_x\partial_y\partial{xy}}
+    \frac{(1+\partial_x^2)\partial_{yy}+(1+\partial_y^2)\partial_{xx}-2\partial_x\partial_y\partial_{xy}}
```
Comment (ojeda-e) [inline membrane_curvature/curvature.py:18]: Thanks for this, that's a hard one to spot.

**Review by ojeda-e — APPROVED** [review]
Thanks!

**Comment by orbeckst** [comment]
You’re welcome. Given that your in charge here, merge it whenever you’re happy with it to go in.

**Pull request merged**